packages/blocks/src/api/test/validation.js

```diff
@@ -662,4 +662,48 @@ describe( 'validation', () => {
 			expect( isValid ).toBe( true );
 		} );
 	} );
+
+	describe( 'validate', () => {
```

Comment:
Should we also test for validation failures?

Reply:
> Should we also test for validation failures?

I don't think it's necessary. The purpose of these test cases is not to verify validation (which are a separate set of test cases), but to ensure the assignment of the validation result.

Reply:
Makes sense.